frontend/src/pages/sites/components/site-card/site-card.tsx

```diff
@@ -0,0 +1,46 @@
+import img from 'src/assets/img/site-mock-image.jpg';
+
+import { Divider, IconButton } from '~/libs/components/components.js';
+
+import styles from './styles.module.scss';
+
+type Properties = {
+  key: number;
+  siteName: string;
```

Comment:
```suggestion
  site: Type
```
pass a full project to props